clients/src/main/java/org/apache/kafka/clients/ClientUtils.java

```diff
@@ -246,13 +247,22 @@ public static NetworkClient createNetworkClient(AbstractConfig config,
         }
     }
 
-    public static <T> List createConfiguredInterceptors(AbstractConfig config,
-                                                        String interceptorClassesConfigName,
-                                                        Class<T> clazz) {
+    public static <T> List getConfiguredInterceptors(AbstractConfig config,
```

Comment:
We tend not to use getters. Could `getConfiguredInterceptors` just be `configuredInterceptors`?